zds/tutorialv2/views/contents.py

```diff
@@ -1871,13 +2001,15 @@ class ContentOfAuthor(ZdSPagingListView):
     template_name = 'tutorialv2/index.html'
     model = PublishableContent
 
+    # list: func filter, Title, Permission, Icon
     authorized_filters = OrderedDict([
-        ('public', [lambda q: q.filter(sha_public__isnull=False), _('Publiés'), True, 'tick green']),
-        ('validation', [lambda q: q.filter(sha_validation__isnull=False), _('En validation'), False, 'tick']),
-        ('beta', [lambda q: q.filter(sha_beta__isnull=False), _('En bêta'), True, 'beta']),
+        ('public', [lambda q, u: q.filter(sha_public__isnull=False, authors__pk__in=[u]), _('Publiés'), True, 'tick green']),
+        ('validation', [lambda q, u: q.filter(sha_validation__isnull=False, authors__pk__in=[u]), _('En validation'), False, 'tick']),
+        ('beta', [lambda q, u: q.filter(sha_beta__isnull=False, authors__pk__in=[u]), _('En bêta'), True, 'beta']),
         ('redaction', [
-            lambda q: q.filter(sha_validation__isnull=True, sha_public__isnull=True, sha_beta__isnull=True),
+            lambda q, u: q.filter(sha_validation__isnull=True, sha_public__isnull=True, sha_beta__isnull=True, authors__pk__in=[u]),
             _('Brouillons'), False, 'edit']),
+        ('testing', [lambda q, u: q.filter(testers__pk__in=[u]), _('À tester'), True, 'search-submit'])
```

Comment:
Actuellement ça prend tous les contenus dont le membre est relecteur, je ne suis pas sûr que ce soit idéal. Comment le relecteur pourra-t-il trier entre les contenus déjà publié et les contenus en cours ? :)

```suggestion
        ('testing', [lambda q, u: q.filter(testers__pk=u), _('À tester'), True, 'search-submit'])
```

Reply:
En effet, peut-être faire une autre liste serait pertinent ?

Reply:
Au lieu de tout mettre ensemble

Reply:
On pourrait avoir deux boutons radios : "Auteur" et "Relecteur" pour pouvoir faire des combinaisons avec "Publié", "En validation", "En bêta" et "Brouillon"

Reply:
Une liste de filtres : "Publié", "En validation", "En bêta" et "Brouillon" avec une autre liste de filtres : "Tous", "Auteur" et "Relecteur" du coup ?

Reply:
Ça me parait être une bonne idée, oui !

Reply:
Je fais ça